README.md

```diff
@@ -63,12 +64,42 @@ objects which can include the optional parameters and values available to [creat
 (`unique`, `multiEntry`, and, for Firefox-only, `locale`). Note that the
 `keyPath` of the index will be set to the supplied index key, or if present,
 a `key` property on the provided parameter object. Note also that when a
-schema is supplied, any object stores not present on the schema object will
-be deleted.
+schema is supplied for a new version, any object stores not present on
+the schema object will be deleted.
 
 A connection is intended to be persisted, and you can perform multiple
-operations while it's kept open. Check out the `/tests/specs` folder
-for more examples.
+operations while it's kept open.
+
+In the event a connection has already been opened for modification (whether
+in the same instance or in another tab/window), a blocking error will occur,
+for which you can listen by adding a `Promise.catch` statement and
+communicate with blocking instances still holding a connection so that
+they may close the connection. You can then return the `resume` property
+(a promise) to recover to continue the original `open` operation and
+proceed to the following `then` condition.
+
+```js
+    var server;
```

Comment:
Since we're using the gated code blocks there doesn't need to be the 4 spaces of indentation first. Can you collapse it back